### PR TITLE
Wizard P3 - imported clusters, storage classes

### PR DESCRIPTION
**Pull request by jruddell**
Part 3 of #1445

Tried a different approach to adding the summary as sometimes the data we want to present is nested and a simple `pick` wont work. Updating the wizard state can probably be cleaned up some as well.

### We need to rebuild some of these flows to use a multi select dropdown when choosing regions. 

image
image
image

### CODE REVIEW

File: src/app/core/components/wizard/WizardSummary.tsx [hunk 1]
```diff
@@ -33,9 +33,21 @@ export default function WizardSummary<T>({
 }: PropsWithChildren<WizardSummaryProps<T>>) {
   const classes = useStyles()
   const allFields = { ...wizardContext, ...fields }
-  const labels = showUndefinedFields
-    ? pickAll(calloutFields as any, allFields)
-    : pickBy(isNotUndefined, pick(calloutFields, allFields))
+
+  // const labels = showUndefinedFields
```
Comment (knnguy) [inline src/app/core/components/wizard/WizardSummary.tsx:37]: comment

File: src/app/plugins/kubernetes/components/app-catalog/deployed-apps/deploy-app-page.tsx [hunk 2]
```diff
@@ -253,16 +253,18 @@ const DeployAppPage = () => {
           context={initialContext}
           submitLabel="Deploy"
           onComplete={handleSubmit}
-          wizardSummaryProps={{
-            icon: <AppIcon appName={appDetail.name} icon={appDetail.metadata?.icon} />,
-            fields: appDetail,
-            keyOverrides: wizardMetaFormattedNames,
-            calloutFields: wizardMetaCalloutFields,
-          }}
+          // wizardSummaryProps={{
```
Comment (knnguy) [inline src/app/plugins/kubernetes/components/app-catalog/deployed-apps/deploy-app-page.tsx:256]: comment

File: src/app/plugins/kubernetes/components/app-catalog/deployed-apps/edit-app-deployment-page.tsx [hunk 3]
```diff
@@ -267,15 +267,17 @@ const EditAppDeploymentPage = () => {
           context={initialContext}
           submitLabel="Deploy"
           onComplete={handleSubmit}
-          wizardSummaryProps={{
-            icon: <AppIcon appName={name} icon={deployedAppDetails?.chart?.metadata?.icon} />,
-            keyOverrides: wizardMetaFormattedNames,
-            calloutFields: wizardMetaCalloutFields,
-          }}
+          // wizardSummaryProps={{
```
Comment (knnguy) [inline src/app/plugins/kubernetes/components/app-catalog/deployed-apps/edit-app-deployment-page.tsx:270]: comment

**Review by knnguy — APPROVED** [review]

File: src/app/plugins/kubernetes/components/app-catalog/deployed-apps/edit-app-deployment-page.tsx (hunk 3 above)
Comment (jruddell) [reply src/app/plugins/kubernetes/components/app-catalog/deployed-apps/edit-app-deployment-page.tsx:270]: I left these in for now as I need to still get the AppIcon to translate to the wizard somewhat dynamically

**Pull request merged**